cfdm/read_write/netcdf/netcdfwrite.py

```diff
@@ -5105,3 +5110,38 @@ def _file_io_iteration(
                 shuffle=shuffle,
                 extra_write_vars=extra_write_vars,
             )
+
+    def _recast_as_int32(self, array):
+        """Recast an array as 32-bit integers.
```

Comment:
I'm going to be pedantic and say that ideally the word 'type'  (or phrase 'data type') can be included here (summary line if it fits, or in the return object description given), just to be fully explicit. (It's clear enough what is meant, though, so you can ignore if you wish.)

Reply:
Or maybe just change to 'recast ... to' since I think 'recast ... as' is potentially ambiguous here?

Reply:
Good points. How about https://github.com/NCAS-CMS/cfdm/pull/162/commits/31c412ea2f372f1349d35b08c5bc6ea88a047f1a ??

Reply:
Perfect, thanks.